src/components/HoverCard/HoverCardOnUsername.tsx

```diff
@@ -0,0 +1,167 @@
+import { useState } from 'react';
+import colors from 'components/core/colors';
+import InteractiveLink from 'components/core/InteractiveLink/InteractiveLink';
+import { BaseM, TitleM } from 'components/core/Text/Text';
+import styled, { css, keyframes } from 'styled-components';
+import { graphql, useFragment } from 'react-relay';
+import { HoverCardOnUsernameFragment$key } from '__generated__/HoverCardOnUsernameFragment.graphql';
+import Markdown from 'components/core/Markdown/Markdown';
+import unescape from 'lodash/unescape';
+import FollowButton from 'components/Follow/FollowButton';
+import transitions, {
+  ANIMATED_COMPONENT_TRANSLATION_PIXELS_LARGE,
+} from 'components/core/transitions';
+import { HoverCardOnUsernameFollowFragment$key } from '__generated__/HoverCardOnUsernameFollowFragment.graphql';
+import { useLoggedInUserId } from 'hooks/useLoggedInUserId';
+
+type Props = {
+  userRef: HoverCardOnUsernameFragment$key;
+  queryRef: HoverCardOnUsernameFollowFragment$key;
+};
+
+export default function HoverCardOnUsername({ userRef, queryRef }: Props) {
+  const [isHovering, setIsHovering] = useState(false);
+
+  const user = useFragment(
+    graphql`
+      fragment HoverCardOnUsernameFragment on GalleryUser {
+        username
+        bio
+        galleries @required(action: THROW) {
+          collections {
+            dbid
+            name
+          }
+        }
+        ...FollowButtonUserFragment
+      }
+    `,
+    userRef
+  );
+
+  const query = useFragment(
+    graphql`
+      fragment HoverCardOnUsernameFollowFragment on Query {
+        ...FollowButtonQueryFragment
+        ...useLoggedInUserIdFragment
+      }
+    `,
+    queryRef
+  );
+
+  const totalCollections = user?.galleries[0]?.collections?.length || 0;
+
+  const handleMouseEnter = () => {
+    setIsHovering(true);
+  };
+
+  const handleMouseLeave = () => {
+    setIsHovering(false);
+  };
+
+  const loggedInUserId = useLoggedInUserId(query);
+  const isLoggedIn = !!loggedInUserId;
+
+  return (
+    <StyledContainer onMouseEnter={handleMouseEnter} onMouseLeave={handleMouseLeave}>
+      <StyledLinkContainer>
+        <InteractiveLink to={`/${user.username}`}>{user.username}</InteractiveLink>
+      </StyledLinkContainer>
+
+      <StyledCardWrapper isHovering={isHovering}>
```

Comment:
also I think the card should be clickable, and navigate to the user's gallery

![Screen Shot 2022-07-26 at 14 56 28](https://user-images.githubusercontent.com/80802871/180935806-e11e6ec1-004b-4a48-b0c4-3b0d17409ec1.png)

Reply:
Added clickable link to the card 🙏🏻